modules/core/src/main/scala/ch/epfl/bluebrain/nexus/commons/search/QueryResult.scala

```diff
@@ -20,15 +21,17 @@ object QueryResult {
     *
     * @param score  the resulting score for the entry
     * @param source the source of the query result
+    * @param sort   the sort parameter of the query result
     */
-  final case class ScoredQueryResult[A](score: Float, source: A) extends QueryResult[A]
+  final case class ScoredQueryResult[A](score: Float, source: A, sort: Option[Seq[Json]]) extends QueryResult[A]
```

Comment:
I would add helpers to build that in a nicer way, since to create a `ScoredQueryResult ` is already convoluted enough

```scala
object ScoredQueryResult {
def apply[A](score: Float, source: A): ScoredQueryResult = ScoredQueryResult(score, source, None)
def apply[A](score: Float, source: A, sort: Seq[Json]): ScoredQueryResult = ScoredQueryResult(score, source, Some(sort))

}
```